UnstoppableWallet/UnstoppableWallet/Core/Managers/RestoreSettingsManager.swift

```diff
@@ -55,7 +55,7 @@ enum RestoreSettingType: String {
         switch self {
```

Comment:
@ant013 This is the parts that I have doubts with. 

I couldn't figure out how to deal with the fact that now the sdk knows both checkpoints and is the app who specifies the network it wants to use.

note: you can't use both at the same time